pyproject.toml

```diff
@@ -75,8 +75,8 @@ Repository = "https://github.com/sainsburyWellcomeCentre/aeon_mecha"
 Documentation = "https://sainsburywellcomecentre.github.io/aeon_docs/"
 DataJoint = "https://docs.datajoint.org/"
 
-[tool.setuptools]
-packages = ["aeon"]
+[tool.setuptools.packages.find]
```

Comment:
Apparently this pattern can also be used. Not sure if one is preferred, / if the * wildcard is necessary?

```
[tool.setuptools.packages.find]
where = ["aeon"]
```

Reply:
```
[tool.setuptools.packages.find]
where = ["src"]  # list of folders that contain the packages (["."] by default)
include = ["my_package*"]  # package names should match these glob patterns (["*"] by default)
exclude = ["my_package.tests*"]  # exclude packages matching these glob patterns (empty by default)
namespaces = false  # to disable scanning PEP 420 namespaces (true by default)
```

[Ref](https://setuptools.pypa.io/en/latest/userguide/pyproject_config.html#:~:text=%5Btool.setuptools.packages,true%20by%20default)

Reply:
I agree `where` probably makes the most sense in this case.

Reply:
Actually as @lochhh pointed out, `where` seems to move the working directory from which to look for packages. However, if we do this, then we lose the actual package folder structure, i.e. the package will not start with `aeon`.

Given this I still think that `include` is most appropriate for now.

Reply:
if we go for `include` we should also specify `exclude = ["docs"]` to exclude `aeon/dj_pipeline/docs`